extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -683,6 +683,10 @@ export class Bind {
   applyBinding_(boundProperty, element, newValue) {
     const property = boundProperty.property;
     switch (property) {
+      case 'value':
```

Comment:
[Bind validator should already support binding to the `value` attribute on input elements](https://github.com/ampproject/amphtml/blob/master/extensions/amp-bind/0.1/bind-validator.js#L323). If this code is removed, does your example break?